### ISSUE TITLE
Port integration tests to Robot Framework

### ISSUE THREAD

**Issue by php-coder** [php-coder/mystamps#530]
After couple attempts to port all integration tests to Thucydides/Serenity/Cucumber-JVM (see #18) I concluded that:
- all work should be merged into the `master` as fast as possible
- probably we shouldn't use only one framework for all integration tests (but we will)

Based on this, the current view is to port most of the integration tests to [Robot Framework](http://robotframework.org) (because the tests are easy to write) ~and port another portion of tests to Cucumber-JVM (later we'll be able to compare them)~. (no, we don't want to maintain 2 frameworks for the same purpose: #1019)

# Plan

## Port the existing tests

- [x] port WhenAdminAddCategory
- [x] port WhenAnonymousUserAddCategory
- [x] port WhenAnonymousUserOpenNotExistingCategoryPage
- [x] port WhenAdminAddCountry
- [x] port WhenAnonymousUserAddCountry
- [x] port WhenAnonymousUserOpenNotExistingCountryPage
- [x] port WhenAdminAddSeries
- [x] port WhenAnonymousUserAddSeries
- [x] port WhenAnonymousUserOpenNotExistingSeriesPage
- [x] port WhenAdminAtIndexPage
- [x] port WhenAnonymousUserActivateAccount
- [x] port WhenAnonymousUserAtForbiddenPage
- [x] port WhenAnonymousUserAtIndexPage
- [x] port WhenAnonymousUserAuthenticates
- [x] port WhenAnonymousUserOpenNotExistingCollectionPage
- [x] port WhenAnonymousUserOpenNotExistingPage
- [x] port WhenAnonymousUserOpenTogglzConsole
- [x] port WhenAnonymousUserRegisterAccount (successfulMessageShouldBeShownAfterRegistration and emailWithActivationKeyShouldBeSentAfterRegistration) (depends on #935)
- [x] port WhenUserAddSeries
- [x] port WhenUserAtActivateAccountPage
- [x] port WhenUserAtAuthPage
- [x] port WhenUserAtIndexPage
- [x] port WhenUserAtRegisterAccountPage
- [x] port WhenUserLogsOut
- [x] port WhenUserOpenTogglzConsole

## Other work
- [x] Integrate [robotframework-lint](https://github.com/boakley/robotframework-lint)
- [x] Document how to use RobotFramework (will be solved in #1021)
- [x] Add link to wiki page to the `Dangerfile`
- [x] Update `NEWS.txt`
- [x] Update `README.md`
- [x] look for leftovers (`git grep When`)
- [x] remove testng/fest-assert from everywhere (Dangerfile, README.md, testng.xml, checkstyle-suppressions.xml, etc)
- [x] Close #18 and do cleanup
- [x] deal with https://github.com/php-coder/mystamps/issues/530#issuecomment-272523664

**Comment by php-coder** [comment]
## Porting of the existing tests
## Other work
- [ ] Integrate [robotframework-lint](https://github.com/boakley/robotframework-lint)
- [ ] Document how to use RobotFramework

**Comment by php-coder** [comment]
Improvements for tests:
- `Create category with name in English`: verify that after changing language, header will be in English
- `Create category with name in English and Russian`: verify that after changing language, header will be in Russian
- `category/creation/logic`: log in as user (instead of admin)

**Comment by php-coder** [comment]
Documented rflint: https://github.com/php-coder/mystamps/wiki/rflint

**Comment by php-coder** [comment]
I think that we done here! 🎉
